assets/index.html

```diff
@@ -1,12 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="UTF-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>Discord Test Client</title>
+	 	<head>
```

Comment:
indentation changed? run `npx prettier -w ./path/to/myChangedFile` from the root dir